query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/schema/RelationOptions.java

```diff
@@ -185,19 +252,30 @@ default SelfT withCompression(@NonNull String compressionAlgorithmName) {
   }
 
   /**
-   * Configures compression using the given algorithm, chunk length and crc check chance.
+   * Configures compression using the given algorithm, chunk length.
    *
    * <p>Unless specifying a custom compression algorithm implementation, it is recommended to use
    * {@link #withLZ4Compression()}, {@link #withSnappyCompression()}, or {@link
    * #withDeflateCompression()}.
    *
    * @param compressionAlgorithmName The class name of the compression algorithm.
    * @param chunkLengthKB The chunk length in KB of compression blocks. Defaults to 64.
-   * @param crcCheckChance The probability (0.0 to 1.0) that checksum will be checked on each read.
-   *     Defaults to 1.0.
    */
   @NonNull
   @CheckReturnValue
+  default SelfT withCompression(@NonNull String compressionAlgorithmName, int chunkLengthKB) {
+    return withOption(
+        "compression",
+        ImmutableMap.of("class", compressionAlgorithmName, "chunk_length_in_kb", chunkLengthKB));
```

Comment:
Does feel a bit weird to use `chunk_length_in_kb` for this implementation, but `chunk_length_kb` for the deprecated one, but this feels like the best way to maintain backwards compatibility for now.  When we can rid of the deprecated methods, `chunk_length_kb` goes away :+1: